YvesIraguha/CodeLab-ReactNative
# #166257426 Setup CircleCI

#### What does this PR do?
- Sets up circleCI 
- Sets up testing environment
#### Description of Task to be completed?
- Create account on circleCI
- Add the repository and configure circleCI
- Configure coveralls and code climate
- Create dummy test  
#### How should this be manually tested?
- Clone the repository and checkout the branch `ch-circle-ci-setup-166257426`
- Install all dependencies by running `yarn` 
- Run tests by running `yarn test` all tests should be passing. 
- Create a new branch and create additional tests. 
- Push the branch to remote and create a PR 
- You should see the report of CirceCI

#### What are the relevant pivotal tracker stories?
#166257426
